src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -327,4 +351,178 @@ public static OpenLoadFlowParameters get(LoadFlowParameters parameters) {
         }
         return parametersExt;
     }
+
+    public static void logDc(LoadFlowParameters parameters, OpenLoadFlowParameters parametersExt) {
+        LOGGER.info("Slack bus selection mode: {}", parametersExt.getSlackBusSelectionMode());
```

Comment:
Maybe you have to add here ` LOGGER.info("Direct current: {}", parameters.isDc());`, no ?

Reply:
Done